packages/components/atoms/f-form-field/src/components/FormLabel.vue

```diff
@@ -103,4 +117,12 @@ input {
         z-index: -1;
     }
 }
+
+.c-formField-label-details {
+    font-weight: $font-weight-regular;
+    color: $form-input-secondaryTextColour;
+    position: absolute;
```

Comment:
Out of interest where does the `position: relative` get set?

Reply:
I think it's here in the fieldWrapper  https://github.com/justeat/fozzie-components/pull/1134/files/e6584f722d4985237459c75b8bdc16f50bb8d7fd#diff-0c855ee5f708709123fe3e0cd37f1060278c6ead61d3a62a053e5c23b45a03b3R385